backend/src/main/java/com/woowacourse/moragora/application/AttendanceService.java

```diff
@@ -109,10 +115,22 @@ public void updateAttendance(final Long meetingId,
     }
 
     public CoffeeStatsResponse countUsableCoffeeStack(final Long meetingId) {
-        final MeetingAttendances meetingAttendances = findMeetingAttendancesBy(meetingId);
-        validateEnoughTardyCountToDisable(meetingAttendances);
-        final Map<User, Long> userCoffeeStats = meetingAttendances.countUsableAttendancesPerUsers();
-        return CoffeeStatsResponse.from(userCoffeeStats);
+        validateMeetingExist(meetingId);
+        final List<Participant> participants = participantRepository.findByMeetingId(meetingId);
+        final int numberOfParticipants = participants.size();
+
+        final PageRequest pageRequest = PageRequest.ofSize(numberOfParticipants);
+        final List<Attendance> attendances = coffeeStatRepository.findCoffeeStatsLimitParticipant(
+                pageRequest, participants);
+
+        validateCoffeeTime(numberOfParticipants, attendances.size());
+
+        final Map<User, Long> coffeeStatsByUser = new HashMap<>();
```

Comment:
```suggestion
        final Map<User, Long> coffeeStacksPerUser = new HashMap<>();
```

여기서도 `CoffeeStack`으로 용어를 통일하고, 한 User '당' coffee stack 개수라는 점에서 `By`보다는 `Per`이라고 표현하는건 어떤가요?

Reply:
좋습니다!!

Reply:
사실 map에는 정확한 naming convention이 없어보이는 것 같아요,

아래와 같은 이름들도 괜찮을 것 같아요

- coffeeStackByUser
- UserToCoffeeStack
- CoffeeStackForUser 

> https://stackoverflow.com/questions/2253453/how-should-i-name-a-java-util-map

Reply:
coffeeStackByUser 로 변경하겠습니다! 참고 문서 감사합니😃